extensions/amazon-aws/src/glue.tsx

```diff
@@ -0,0 +1,291 @@
+          <ActionPanel.Section title={"Copy"}>
+            <Action.CopyToClipboard title="Copy Job Name" content={glueJobRun.JobName || ""} />
+          </ActionPanel.Section>
+        </ActionPanel>
+      }
+      accessories={[
+        { date: glueJobRun.CompletedOn, tooltip: "Completed On" },
+        {
+          text: glueJobRun.accessoriesText,
+          icon: { source: glueJobRun.icon!, tintColor: glueJobRun.iconTintColor },
+          tooltip: "Recent Job Run State",
+        },
+      ]}
+    />
+  );
+}
+
+function GlueJobRuns({ glueJobName: glueJobName }: { glueJobName: string }) {
+  const { data: glueJobRuns, isLoading: isLoadingJobRuns } = useCachedPromise(fetchJobRuns, [25, [glueJobName]]);
+  return (
+    <List isLoading={isLoadingJobRuns} navigationTitle={`Glue Job Runs of ` + glueJobName} isShowingDetail>
+      {glueJobRuns?.map((jobRun) => (
+        <List.Item
+          accessories={[
+            { date: jobRun.CompletedOn, tooltip: "Completed On" },
+            {
+              text: jobRun.accessoriesText,
+              icon: { source: jobRun.icon!, tintColor: jobRun.iconTintColor },
+              tooltip: "Recent Job Run State",
+            },
+          ]}
+          title={jobRun.CompletedOn?.toUTCString() || ""}
+          detail={<GlueJobRunDetails jobRun={jobRun} />}
+          actions={
+            <ActionPanel>
+              <AwsAction.Console url={resourceToConsoleLink(jobRun.Id, "AWS::Glue::JobRun")} />
+              <ActionPanel.Section title={"Copy"}>
+                <Action.CopyToClipboard title="Copy Job Run Id" content={jobRun.Id || ""} />
+              </ActionPanel.Section>
+            </ActionPanel>
+          }
+        />
+      ))}
+    </List>
+  );
+}
+
+function GlueJobRunDetails({ jobRun: glueJobRun }: { jobRun: GlueJobRun }) {
+  return (
+    <List.Item.Detail
+      metadata={
+        <List.Item.Detail.Metadata>
+          <List.Item.Detail.Metadata.Label title="Job Name" text={glueJobRun.JobName} />
+          <List.Item.Detail.Metadata.Label
+            title="Job Run State"
+            text={{ value: glueJobRun.JobRunState || "", color: glueJobRun.iconTintColor }}
+          />
+          <List.Item.Detail.Metadata.Label title="Execution Time" text={glueJobRun.ExecutionTime?.toString() + "s"} />
+          <List.Item.Detail.Metadata.Label title="Started on" text={glueJobRun.StartedOn?.toUTCString()} />
+          <List.Item.Detail.Metadata.Label title="Completed On" text={glueJobRun.CompletedOn?.toUTCString() || ""} />
+          <List.Item.Detail.Metadata.Label
+            title="Error Message"
+            text={{ color: Color.Red, value: glueJobRun.ErrorMessage || "" }}
+          />
+          <List.Item.Detail.Metadata.Label title="Arguments" text={JSON.stringify(glueJobRun.Arguments)} />
+          <List.Item.Detail.Metadata.Label title="DPU Seconds" text={(glueJobRun.DPUSeconds?.toString() || "") + "s"} />
```

Comment:
For optional properties like these, please render them optionally, for example:

https://github.com/raycast/extensions/blob/b93225144bde66a7433401e2b4b6270538fa2265/extensions/amazon-aws/src/dynamodb.tsx#L167-L169

Otherwise you would be left with empty titles like in the screenshot.